app/gui/view/debug_scene/cloud-dashboard/cloud-http/src/lib.rs

```diff
@@ -0,0 +1,200 @@
+#![warn(explicit_outlives_requirements)]
+#![warn(missing_copy_implementations)]
+#![warn(missing_debug_implementations)]
+#![warn(noop_method_call)]
+#![warn(single_use_lifetimes)]
+#![warn(trivial_casts)]
+#![warn(trivial_numeric_casts)]
+#![warn(unused_extern_crates)]
+#![warn(unused_import_braces)]
+#![warn(unused_lifetimes)]
+#![warn(unused_qualifications)]
+#![warn(variant_size_differences)]
+#![warn(unreachable_pub)]
+
+use enso_prelude::*;
+use enso_cloud_view::prelude::*;
+
+use response::Route;
+
+
+// ==============
+// === Export ===
+// ==============
+
+pub mod response;
+
+
+
+// ==============
+// === Client ===
+// ==============
+
+/// Client used to make HTTP requests to the Cloud API.
+/// 
+/// This struct provides convenience functions like [`Route::list_projects`] which let you send HTTP
+/// requests to the Cloud API without building HTTP requests manually or dealing with details like
+/// adding HTTP headers for authorization, etc. The convenience functions return strongly-typed
+/// responses that are automatically deserialized from JSON.
+#[derive(Clone, Debug)]
+pub struct Client {
+    /// The URL that Cloud API requests are sent to, minus the relative path to an API endpoint.
+    /// 
+    /// [`Route`]s appended to this URL as a relative path to form the full URL of a request. Each
+    /// API endpoint has an associated [`Route`].
+    /// 
+    /// This URL is constructed differently for different environments. For example, in production
+    /// this URL is usually our public domain (e.g., `https://cloud.enso.org`), in staging it is an
+    /// AWS API Gateway URL (e.g., `https://12345678a.execute-api.us-west-1.amazonaws.com`), and in
+    /// development it is a local URL (e.g., `http://localhost:8080`).
+    /// 
+    /// If you are developing against a deployed environment, the AWS API Gateway URL can be
+    /// looking in the Terraform output logs after a successful deployment.
+    base_url: String,
+    /// The JSON Web Token (JWT) used to authenticate requests to our API.
+    token:    AccessToken,
+    /// Underlying HTTP client used to make requests.
+    /// 
+    /// This struct wraps the HTTP client and hides the details of how requestts are made, so that
+    /// we can provide a simpler API to users of this crate. For example, use the
+    /// [`Client::list_projects`] method to make requests to the [`Route::ListProjects`] endpoint.
+    http:     reqwest::Client,
+}
+
+
+// === Main `impl` ===
+
+impl Client {
+    /// Creates a new instance of the Cloud API [`Client`].
+    /// 
+    /// For more info about how the arguments are used, see the documentation of the [`Client`]
+    /// struct and its fields, or the type documentation for the arguments.
+    pub fn new(
+        api_gateway_id: ApiGatewayId,
+        aws_region: AwsRegion,
+        token: AccessToken,
+    ) -> Self {
+        let base_url = format!("https://{api_gateway_id}.execute-api.{aws_region}.amazonaws.com");
```

Comment:
imho its better to have base_url as a parameter. We will move from apig generated api addresses in few days (the main will be api.cloud.enso.org so personal ones can be api.npekin.cloud.enso.org). This way you can set url as a env variable